lib/spack/spack/cmd/create.py

```diff
@@ -328,65 +353,169 @@ def __call__(self, stage, url):
 
         # Determine the build system based on the files contained
         # in the archive.
-        build_system = 'unknown'
+        build_system = 'default'
```

Comment:
I like the idea of just special casing the method to something like "couldn't detect a build system -- using generic package", since that gives the user more information anyway.